mm2src/crypto/src/bip32_child.rs

```diff
@@ -8,23 +9,21 @@ use std::str::FromStr;
 pub type HardenedValue = AnyValue<true>;
 pub type NonHardenedValue = AnyValue<false>;
 
-#[derive(Debug, Eq, PartialEq)]
+#[derive(Debug, Eq, PartialEq, Display)]
```

Comment:
If the above comments are fixed we might not need this Display

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2432/commits/b4531266eae378da1ea48d0a6aacad0efc3bc67b